src/Services/CurrencyDisplayService.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+require_once './src/Entities/Product.php';
+
+class CurrencyDisplayService
```

Comment:
missing tests for this class

Reply:
also not a great class name. it doesnt actually display anything, it just converts a number into a currency